supernode/services/artworkregister/task.go

```diff
@@ -796,6 +823,28 @@ func (task *Task) storeThumbnails(ctx context.Context) error {
 	return nil
 }
 
+func (task *Task) storeIDFiles(ctx context.Context) error {
+	store := func(ctx context.Context, ids []string) error {
+		for _, idFile := range ids {
+			if _, err := task.p2pClient.Store(ctx, []byte(idFile)); err != nil {
+				return err
+			}
+		}
+
+		return nil
+	}
+
+	if err := store(ctx, task.Ticket.AppTicketData.DDAndFingerprintsIDs); err != nil {
```

Comment:
I don't think you should store DDAndFingerprintsIDs into p2p, should store : 
`compressed(Base64(dd_and_fingerprints).Base64(signatureSN1).Base64(signatureSN2).Base64(signatureSN3).1234)`